current/en-us/2. tutorials/2. creating-a-contact-manager.md

```diff
@@ -754,14 +754,14 @@ Previously, when we created the `contact-list` component, we required that into
 import {PLATFORM} from 'aurelia-framework';
 
 export function configure(config) {
-  config.globalResources([PLATFORM.moduleName('./elements/loading-indicator')]);
+  config.globalResources([PLATFORM.moduleName('./loading-indicator')]);
```

Comment:
@bigopon Is this a possible difference between Webpack and Require.js or will it work the same in both cases?

Reply:
Note that I tested this with TypeScript - not with ES